src/kOS/Suffixed/Direction.cs

```diff
@@ -194,6 +194,12 @@ public override bool Equals(object obj)
             return false;
         }
 
+        // Needs to be overwritten because Equals() is overridden.
+        public override int GetHashCode()
+        {
+            return rotation.GetHashCode();
+        }
+
```

Comment:
Fixes warning: This class overrides Equals() but doesn't override GetHashCode().

Since this type (Direction) can be put into a Lexicon by the user, having the hashcode match the meaning of Equals could be important.